src/link/index.tsx

```diff
@@ -47,3 +47,6 @@ export const StyledLink = withWrapper<typeof Link, { animateUnderline?: boolean
       );
     }
 );
+
+/** @deprecated To be removed in future versions.*/
+export type LinkProps = React.ComponentProps<typeof StyledLink>;
```

Comment:
I couldn't find a reference to this, is it required?

Reply:
it is for type that was exported here - https://github.com/uber/baseweb/blob/master/legacy-dts/link/index.d.ts#L4-L11